frontend/public/components/factory/table.tsx

```diff
@@ -104,12 +105,38 @@ const sorts = {
   serviceCatalogStatus,
   jobCompletions: (job) => getJobTypeAndCompletions(job).completions,
   jobType: (job) => getJobTypeAndCompletions(job).type,
+  managedNamespace: (obj) => {
```

Comment:
Is there any way to contribute these specific sorts from OLM through extensions? if not we should consider adding something.

Reply:
It looks like operand in operator-lifecycle-manager package isn't listing its sort here and keeps it within the package instead; I can go with that approach if that works.